README.md

```diff
@@ -50,41 +50,38 @@ Voraussetzung ist ein Linux System mit installiertem python 3.8
 #### Architektur-Überblick
 Der Server wird durch eine Flask App (`/app`) implementiert. Diese verwaltet verschiedene API-Endpunkte (`/app/routes`). Außerdem wird eine Datenbank verwaltet. Deren ORM wird in `/app/models` implementiert. Der Story-Inhalt und der Story-verwaltende Code liegt unter `/app/story`.  
 
-#### Nutzung ohne App
-Unsere App ist sehr datenhungrig. Um das Spiel zu testen, ohne die App zu installieren, haben wir ein Postman package erstellt. Dieses immitiert die App. Damit lässt sich das Spiel auch ohne Appnutzung durchspielen (die Spielerfahrung leidet darunter jedoch deutlich). Die Nutzung ist wie folgt:
+#### Spielen ohne Freigeben persönlicher Daten
+Unsere App ist sehr datenhungrig. Deswegen startet die App per default in einem Sicherheitsmodus, der das Senden persönlicher Daten verhindert.
+Um das Spiel trotzdem spielen zu können, haben wir ein Postman package erstellt. Dieses bietet Mock-Daten an, welche die sonst von der App gestohlenen Daten ersetzen.
 1. `detective-game-no-app-walkthrough.json` mit Postman **Desktop** importieren
-1. `remote_testing_environment.json` bzw. `localhost_testing_environment.json` importieren
-   (je nachdem, ob der Server auf einer anderen Rechner läuft oder nicht)
-1. `create user` senden, um einen neuen Nutzer zu erstellen
-1. die URL aus der response öffnen, um den Chat mit Kommissar Rex zu starten
+1. Die Postman Umgebungsvariable `server_ip` auf die Adresse des Rechners setzten, auf dem die Serverkomponente (der Code dieses Repos) läuft
+1. Die App starten und *"Kommissar Rex kontaktieren"* tippen und in Telegram auf `STARTEN` tippen
```

Comment:
This does not seem correct, do you mean `/start`?

Reply:
no, if Telegram is opened via the App, there is a big Button instead of the normal text input that says "STARTEN". `/start` is the command that is sent when you click this button, but it's not the text on the initial button